packages/yew-macro/tests/classes_macro/classes-fail.stderr

```diff
@@ -21,7 +21,7 @@ error[E0277]: the trait bound `Classes: From<{integer}>` is not satisfied
             <Classes as From<&Option<T>>>
             <Classes as From<&String>>
             <Classes as From<&[T]>>
-            <Classes as From<&implicit_clone::unsync::IString>>
+            <Classes as From<&implicit_clone::unsync::string::IString>>
```

Comment:
@kirillsemyonkin I think this is an unfortunate consequence of the recent change of module architecture in implicit-clone

I don't mind too much keeping this but I would have prefer to avoid making things uselessly verbose like that.

Related to https://github.com/yewstack/implicit-clone/pull/33

Reply:
User uses it via `implicit_clone::unsync::IString`. The fact that Rust does not show usable path and shows an implementation detail is Rust's problem. `implicit_clone::unsync::string::IString` is fine to me though, since it kind of mirrors `std::string::String`.

Reply:
I don't mind too much so we can keep it that way.

But I would like to point out that in order to improve "correctness" we have degraded the user experience of the crate. I hope this will make us reflect on the relevance of these sacred "good Rust" patterns, and "code quality".

Reply:
And focus on what matters the most: user's experience.

Reply:
I mean dev experience is good too (more or less important is arguable), and it didn't work in VSCode (rust-analyzer) before, but now it does work properly, which means to edit IString for example I do not need to do it blindly with a rust doc open on std and even syntax instead of simple suggestions and real time linting

Reply:
Ah yeah I forgot about that issue with rust-analyzer sorry (I don't use this). So I guess yes there is some improvement somewhere in the end. Though I personally do use rustc messages A LOT and for me it's a loss.

(Also you said "Rust's problem" but I could argue that this is rust-analyzer's problem too. Not that I want to be picky but I think there are both side to see here. Different people, different work environment)